pkg/sql/create_user.go

```diff
@@ -38,6 +41,13 @@ type createUserNode struct {
 //   notes: postgres allows the creation of users with an empty password. We do
 //          as well, but disallow password authentication for these users.
 func (p *planner) CreateUser(ctx context.Context, n *tree.CreateUser) (planNode, error) {
+	return p.CreateUserNode(ctx, n.Name, n.Password, n.IfNotExists, false /* isRole */, "CREATE USER")
+}
+
+// CreateUserNode creates a user node. This can be called from CREATE USER or CREATE ROLE.
```

Comment:
nit: either say `creates a CreateUserNode` or `creates a create user plan node`.

Reply:
Done. And added a similar comment on `DropUserNode`. I'm surprised lint didn't yell at me.